lib/components/form/user-settings.js

```diff
@@ -65,6 +63,24 @@ function formatElapsedTime(timestamp, intl) {
   return intl.formatMessage({ id: 'common.time.durationAgo' }, { duration })
 }
 
+/**
+ * Helper to ensure the home and work locations are correctly populated,
+ * or a placeholder is created.
+ */
+function addOrEditPlaceIfNeeded(locations, type, icon) {
+  const editedLocation = locations.find((loc) => loc.type === type)
+  if (editedLocation) {
+    if (!editedLocation.address) editedLocation.address = editedLocation.name
```

Comment:
What's the purpose of setting the address to equal the name? Can you add a comment explaining why we do this?

Reply:
Good point, I added the comment in https://github.com/opentripplanner/otp-react-redux/pull/957/commits/457cf905218b8195b3a0fcaf9dcae5299342b468.